packages/p/pokeget/package.yml

```diff
@@ -0,0 +1,20 @@
+name       : pokeget
+version    : 1.6.3
+release    : 1
+source     : 
+    - git|https://github.com/talwat/pokeget-rs.git : 1.6.3
```

Comment:
Any reason you're using git?

Reply:
This is missing in the tarball https://github.com/msikma/pokesprite/tree/c5aaa610ff2acdf7fd8e2dccd181bca8be9fcb3e